docs/source/markdown/podman-volume-create.1.md

```diff
@@ -129,6 +129,80 @@ All containers are assigned larger project IDs (e.g. >= 100000).
 All volume assigned project IDs larger project IDs starting with 200000.
 This prevents xfs_quota management conflicts with containers/storage.
 
+## MOUNT EXAMPLES
+
+podman volume create allows you to pass the `type`, `device`, and `o` options to `mount(8)` when using the `local` driver.  
+
+Here are some examples of what you can do with that
```

Comment:
Given the prominent heading `MOUNT EXAMPLES`, I'm not sure why/if these two lines are needed.

Reply:
I concur, I'd drop line 136.

Reply:
Wanted to separate it from the other examples, like previously mentioned, this is a library that is not controlled by this project, but because it can interface with `mount` it can integrate with other libraries.  
Wanted to also have a new heading for future expansion.  

Some ideas:
Maybe someone figures something out with that shows off a capability, like: mounting a [iSCSI](https://github.com/sahlberg/libiscsi) target, an [iPFS](https://github.com/ipfs/kubo/blob/master/docs/fuse.md) target or mounting a weird device like USB for use with USB/IP, an example with using [bindfs](https://bindfs.org/) that allows you to do something cool with mirrors, maybe mounting [archives](https://github.com/google/fuse-archive)  

---

If it is still out of place, where would you like me to put it?  
Under the existing examples?

Reply:
Ah wait, I re-read your comment, I missunderstood what you meant. I agree, it doesn't read very nice.